std/numeric.d

```diff
@@ -12,7 +12,7 @@ WIKI = Phobos/StdNumeric
 Copyright: Copyright Andrei Alexandrescu 2008 - 2009.
 License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
 Authors:   $(WEB erdani.org, Andrei Alexandrescu),
-                   Don Clugston, Robert Jacques
+                   Don Clugston, Robert Jacques, Ilya Yaroshenko
```

Comment:
#3559 (findLocalMin), #2500 (sumOfLog2s), and `findRoot` fixes and optimizations: #3028, #2639, #2524, #2266